### DISCUSSION TITLE
Adjustable scale at which house number is shown

### DISCUSSION THREAD

**Discussion by ihareg** [streetcomplete/StreetComplete#5529] [Ideas]
I use the app to update house numbers in locations which I attend. And it's always inconvenient to adjust scale manually to find which house number building has,  it appears on very low scale. Is it possible to make a scale at which house numbers appear more higher or to make it adjustable in the app settings. "helicopter" view will allow to re-check more addresses than zoom-in and zoom-out every house in locality. Much appreciate if it's possible to do something with it.

**Comment by matkoniecz** [comment]
Have you tried enabling address overlay? It shows addresses for a bit longer.
